src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+
+  if (days < SHORT_TERM) {
+    return days * PRICE_PER_DAY;
+  }
+
+  if (days < LONG_TERM) {
+    return days * PRICE_PER_DAY - SHORT_TERM_DISCOUNT;
```

Comment:
Consider adding a check to ensure that 'days' is a positive number before proceeding with the calculation. This will handle potential cases where a negative or zero value is passed to the function.